controller/api/api.go

```
@@ -612,10 +612,14 @@ func (a *Api) scaleContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := a.manager.ScaleContainer(containerId, numInstances); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	result := a.manager.ScaleContainer(containerId, numInstances)
+	json, _ := json.Marshal(result)
```

I prefer to catch errors rather than swallow :)